From 13b2d96ee226ffaa487280c4b454346e86a76efc Mon Sep 17 00:00:00 2001
From: Pierre-Louis Bonicoli <pierrelouis.bonicoli@scality.com>
Date: Wed, 13 Jan 2016 09:14:13 +0100
Subject: [PATCH 07/12] Inline short method used once

Issue: RELENG-338
---
 wall_e.py | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/wall_e.py b/wall_e.py
index 3068216..53ed5af 100644
--- a/wall_e.py
+++ b/wall_e.py
@@ -226,9 +226,6 @@ class IntegrationBranch(Branch):
             raise Conflict(source=source_branch,
                            destination=self)
 
-    def merge_from_development_branch(self):
-        self.merge_from_branch(self.development_branch)
-
     def check_history_did_not_change(self):
         feature_branch = FeatureBranchName(self.subname)
         for commit in self.get_all_commits_since_started_from(feature_branch):
@@ -442,7 +439,7 @@ class WallE:
         # that are not in development/* or in the feature branch.
         self.integration_branches[0].check_history_did_not_change()
         for integration_branch in self.integration_branches:
-            integration_branch.merge_from_development_branch()
+            integration_branch.merge_from_branch(integration_branch.development_branch)
 
     def update_integration_branches_from_feature_branch(self):
         branch_to_merge_from = self.source_branch
-- 
2.7.0.rc3

