From 6a6dd811934320b6d61e9a2ff5060415877dff3a Mon Sep 17 00:00:00 2001
From: Pierre-Louis Bonicoli <pierrelouis.bonicoli@scality.com>
Date: Wed, 13 Jan 2016 09:31:16 +0100
Subject: [PATCH 11/12] new test: validate branch names without calling a web
 service

---
 test_wall_e.py | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/test_wall_e.py b/test_wall_e.py
index 768aad0..66c43ee 100644
--- a/test_wall_e.py
+++ b/test_wall_e.py
@@ -73,6 +73,16 @@ def rebase_branch(repo, branch_name, on_branch):
     repo.cmd('git push -f')
 
 
+class QuickTest(unittest.TestCase):
+    """Tests which don't need to interact with an external web services"""
+
+    def test_refuse_feature_on_maintenance_branch(self):
+        src = wall_e.FeatureBranchName('feature/RING-0004')
+        dest = wall_e.DestinationBranchName('development/4.3')
+        with self.assertRaises(BranchDoesNotAcceptFeatures):
+            src.check_if_should_handle(dest)
+
+
 class TestWallE(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
-- 
2.7.0.rc3

