From 70002f261a0a6e29b2622a70bd48ccfc6eee57c7 Mon Sep 17 00:00:00 2001
From: Pierre-Louis Bonicoli <pierrelouis.bonicoli@scality.com>
Date: Wed, 13 Jan 2016 10:14:10 +0100
Subject: [PATCH 12/12] wip

---
 git_api.py | 6 +++---
 wall_e.py  | 4 ++--
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/git_api.py b/git_api.py
index 0626dd6..7659518 100644
--- a/git_api.py
+++ b/git_api.py
@@ -86,14 +86,14 @@ class Branch(object):
             self.push() # FIXME push will do an unnecessary checkout
 
     def get_all_commits_since_started_from(self, source_branch):
-        source_branch.checkout()
-        log = cmd('git log --no-merges --pretty=%%H %s..%s' % (
+        self.repo.checkout(source_branch.name)
+        log = self.repo.cmd('git log --no-merges --pretty=%%H %s..%s' % (
             source_branch.name, self.name))
         return log.splitlines()
 
     def includes_commit(self, sha1):
         try:
-            cmd('git merge-base --is-ancestor %s %s' % (sha1, self.name))
+            self.repo.cmd('git merge-base --is-ancestor %s %s' % (sha1, self.name))
         except subprocess.CalledProcessError:
             return False
         return True
diff --git a/wall_e.py b/wall_e.py
index b3b9d67..cfaef21 100644
--- a/wall_e.py
+++ b/wall_e.py
@@ -644,14 +644,14 @@ class WallE:
             match_ = re.match(regexp, raw_cleaned)
             if not match_:
                 logging.warning('Command comment ignored. '
-                                'Unknown format: %s' % raw)
+                                'Unknown format: %s', raw)
                 continue
 
             command = match_.group('command')
 
             if not self.check_command(author, command):
                 logging.warning('Command comment ignored. '
-                                'Checks failed: %s' % raw)
+                                'Checks failed: %s', raw)
                 continue
 
             # get command handler and execute it
-- 
2.7.0.rc3

